core/metacling/src/TClingMethodInfo.cxx

```diff
@@ -478,13 +478,13 @@ long TClingMethodInfo::Property() const
    }
    switch (Access) {
       case clang::AS_public:
-         property |= kIsPublic;
+         property |= kIsPublic | kIsNotReacheable;
```

Comment:
Should this move down to the `clang::AS_protected` case?

Reply:
Indeed.